docs/deploy-guide/getting-started.md

```diff
@@ -2,20 +2,70 @@
 
 You will need to have available a number of local utilities,
 a clone of this repo and another Git repo which will be referred to
-as the deploy repo. You will also need at least once Kubernetes
-cluster available to you, while multiple clusters are the advisable
-approach for any production or deployment at scale as the [Introduction](./index.md)
-mentions.
+as the deploy repo. You will also need at least one Kubernetes
+cluster available to you, which can be an All-In-One deployment,
+while multiple clusters are the advisable approach, as described
+in the [System Division](./welcome.md#system-division),
+for any production or deployment at scale.
```

Comment:
I will rephrase this to something like, it's quick to understand things in points.

---

To get started, ensure you have the following prerequisites:
* A clone of this repository.
* Another [Git repository,](https://github.com/RSS-Engineering/undercloud-deploy) referred to as the deploy repo.
* Access to at least one Kubernetes cluster.

For simplicity, you can use an All-In-One deployment. However, for production or large-scale deployments, it is recommended to use multiple clusters, as outlined in the System Division.